ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -0,0 +1,120 @@
+package com.revenuecat.purchases.ui.revenuecatui.data
+
+import androidx.compose.ui.res.stringResource
+import com.revenuecat.purchases.Offering
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.PackageType
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.Period
+import com.revenuecat.purchases.models.Price
+import com.revenuecat.purchases.models.PurchasingData
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.SubscriptionOption
+import com.revenuecat.purchases.models.SubscriptionOptions
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.net.URL
+
+class TestData {
+    companion object {
```

Comment:
Instead of creating a companion object, since everything here is "static", you could just make `TestData` an `object`, like 
```
internal object TestData {
   ...
}
```

Same for the `Packages` and `Constants` classes

Reply:
Thank you! I'm still learning Kotlin, so this feedback is very welcome.